bjoern.py

```diff
@@ -35,11 +35,13 @@ def bind_and_listen(host, port=None, reuse_port=False,
     return sock
 
 
-def server_run(sock, wsgi_app):
-    _bjoern.server_run(sock, wsgi_app)
+def server_run(sock, wsgi_app, statsd=None):
+    if statsd is None:
+        _bjoern.server_run(sock, wsgi_app)
+    else:
+        _bjoern.server_run(sock, wsgi_app, int(statsd['enable']), statsd['host'], statsd['port'], statsd['ns'], statsd.get('tags'))
```

Comment:
it forces users to declare all statsd options even if they want to use the defaults. Is this intended?

Reply:
Yes, I don't want any statsd default options inside the bjoern code. These values are completely unrelated to bjoern. For example if the statsd defaults change in future (e.g. they change to a different port) then we shouldn't have to make any changes to bjoern.

Also, it's just 4 values, and it forces the caller code to be explicit rather than implicit, which is always better :)